manim/utils/config_utils.py

```diff
@@ -521,10 +508,72 @@ def finalized_configs_dict():
     return {section: dict(config[section]) for section in config.sections()}
 
 
-def _subcommands_exist(ignore=[]):
+def _subcommand_name(ignore=[]):
+    """Goes through sys.argv to check if any subcommand has been passed,
+    and returns the first such subcommand's name, if found.
+
+    Parameters
+    ----------
+    ignore : Optional[:class:`list`]
```

Comment:
list of what? also, is it ok to specify a tuple instead? so we shouldnt restrict this to just list
do we need it to be iterable? sure, Iterable[<type>]
Do we need it to be indexable (i.e., to be able to `ignore[i]`)? Sure, Sequence[<type>]

Also note that `Optional[<x>]` means it can be either `<x>` or `None`, while `<x>, optional` means that the param can or can not be specified (i.e., has a default value). I'm sure you don't want `None` here, right?

Judging from the code...:
```suggestion
    ignore : Iterable[:class:`str`], optional
```